### PR TITLE
GBI: Minor cleanup

### PR DESCRIPTION
**Pull request by lioncash**

### CODE REVIEW

**Comment by gonetz** [comment]
Very clean and accurate commits. The code definitely looks better with them. Thanks!

File: src/GBI.cpp [hunk 1]
```diff
@@ -142,6 +143,11 @@ bool GBIInfo::isHWLSupported() const
 	return true;
 }
 
+void GBIInfo::_flushCommands()
```
Comment (Narann) [inline src/GBI.cpp]: What about using `constexpr`?

Comment (lioncash) [inline src/GBI.cpp]: std::fill does not satisfy the constexpr requirements.

Comment (Narann) [inline src/GBI.cpp]: I wonder if the original version couldn't be `constexpr`'ed:
```cpp
constexpr void GBIInfo::_flushCommands()
{
    for (u32 i = 0; i <= 0xFF; ++i)
        cmd[i] = GBI_Unknown;
}
```

Comment (lioncash) [inline src/GBI.cpp]: Sure, but MSVC doesn't support extended constexpr, which is why I didn't do that.

**Pull request merged**